dds/src/implementation/dds_impl/domain_participant_impl.rs

```diff
@@ -903,78 +895,35 @@ impl DdsShared<DomainParticipantImpl> {
-            self.get_domain_tag(),
-        ) {
-            if !self.is_participant_ignored(discovered_participant_data.get_serialized_key().into())
-            {
-                self.get_builtin_publisher()
-                    .sedp_builtin_publications_writer()
-                    .add_matched_participant(&participant_discovery);
-
-                self.get_builtin_subscriber()
-                    .sedp_builtin_publications_reader()
-                    .add_matched_participant(&participant_discovery);
-
-                self.get_builtin_publisher()
-                    .sedp_builtin_subscriptions_writer()
-                    .add_matched_participant(&participant_discovery);
-
-                self.get_builtin_subscriber()
-                    .sedp_builtin_subscriptions_reader()
-                    .add_matched_participant(&participant_discovery);
-
-                self.get_builtin_publisher()
-                    .sedp_builtin_topics_writer()
-                    .add_matched_participant(&participant_discovery);
-
-                self.get_builtin_subscriber()
-                    .sedp_builtin_topics_reader()
-                    .add_matched_participant(&participant_discovery);
-
-                self.discovered_participant_add(
-                    discovered_participant_data.get_serialized_key().into(),
-                    discovered_participant_data,
-                );
-            }
-        }
-    }
-
-    pub fn remove_discovered_participant(&self, participant_handle: InstanceHandle) {
-        if let Some((_, discovered_participant_data)) = self
-            .discovered_participant_list()
-            .into_iter()
-            .find(|&(h, _)| h == &participant_handle)
-        {
-            let participant_guid_prefix = discovered_participant_data.guid_prefix();
-            self.get_builtin_subscriber()
-                .sedp_builtin_publications_reader()
-                .remove_matched_participant(participant_guid_prefix);
-            self.get_builtin_subscriber()
-                .sedp_builtin_subscriptions_reader()
-                .remove_matched_participant(participant_guid_prefix);
-            self.get_builtin_subscriber()
-                .sedp_builtin_topics_reader()
-                .remove_matched_participant(participant_guid_prefix);
-            self.get_builtin_publisher()
-                .sedp_builtin_publications_writer()
-                .remove_matched_participant(participant_guid_prefix);
-            self.get_builtin_publisher()
-                .sedp_builtin_subscriptions_writer()
-                .remove_matched_participant(participant_guid_prefix);
-            self.get_builtin_publisher()
-                .sedp_builtin_topics_writer()
-                .remove_matched_participant(participant_guid_prefix);
-        }
-
-        self.discovered_participant_remove(participant_handle);
-    }
+    // pub fn remove_discovered_participant(&self, participant_handle: InstanceHandle) {

```

Comment:
So it's not tested?

Reply:
It was used before by the ignore but not explicitly verified. So it isn't really tested